rest/resource-server/src/docs/asciidoc/schedule.adoc

```diff
@@ -22,4 +23,70 @@ A `DELETE` request will cancel all the services.
 include::{snippets}/should_document_cancel_all_schedule/curl-request.adoc[]
 
 ===== Example response
-include::{snippets}/should_document_cancel_all_schedule/http-response.adoc[]
\ No newline at end of file
+include::{snippets}/should_document_cancel_all_schedule/http-response.adoc[]
+
+[[schedule-cve]]
+==== Schedule cve service
+
+A `POST` request will schedule the cve service.
+
+===== Example request
+include::{snippets}/should_document_schedule_cve_service/curl-request.adoc[]
+
+===== Example response
+include::{snippets}/should_document_schedule_cve_service/http-response.adoc[]
+
+[[unschedule-cve]]
+==== Unschedule cve search
+
+A `DELETE` request will unschedule the cve search.
+
+===== Example request
+include::{snippets}/should_document_unschedule_cve_search/curl-request.adoc[]
+
+===== Example response
+include::{snippets}/should_document_unschedule_cve_search/http-response.adoc[]
+
+[[schedule-service]]
+==== Schedule service for attachment deletion from local FS.
+
+A `POST` request will schedule attachment deletion.
+
+===== Example request
+include::{snippets}/should_document_schedule_service_from_local/curl-request.adoc[]
+
+===== Example response
+include::{snippets}/should_document_schedule_service_from_local/http-response.adoc[]
+
+[[cancel-schedule]]
+==== Cancel schedule attachment from local fs.
+
+A `DELETE` request will schedule attachment deletion.
+
+===== Example request
+include::{snippets}/should_document_cancel_schedule_attachment/curl-request.adoc[]
+
+===== Example response
+include::{snippets}/should_document_cancel_schedule_attachment/http-response.adoc[]
+
+[[delete-attachment]]
+==== Delete old attachment from local fs.
```

Comment:
remove `old` as I don't see such word in UI

Reply:
done.